spec/models/claim_spec.rb

```diff
@@ -905,14 +905,15 @@
   end
 
   describe "Early Career Payments claim" do
-    let(:eligibility) { build(:early_career_payments_eligibility) }
+    let(:eligibility) { build(:early_career_payments_eligibility, :eligible) }
 
     describe "#submittable?" do
```

Comment:
Should be in context under existing describe for `#submittable?`.

Reply:
Done as suggested. Thanks for pointing out the error in my logic and positioning @nomasprime